### PR TITLE
Feature/tenant model/flovin

### PR DESCRIPTION
**Pull request by deguzmankevin**
This PR adds the tenant model and its association with a referral agency!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,15 +4,19 @@ services:
     image: postgres:10-alpine
     environment:
       - POSTGRES_USER=root
+    privileged: true
```
Comment (deguzmankevin) [inline docker-compose.yml]: when I talked to Ethan I'm pretty sure he said these docker-compose changes shouldn't affect Mac

**Comment by deguzmankevin** [comment]
checked with Ethan and got my docker stuff fixed! he said it's fine to merge these new changes into master

File: db/migrate/20181011035806_create_tenants.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class CreateTenants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :tenants do |t|
+      t.string :name
+      t.text :description
+      t.string :email
+      t.string :phone
+      t.string :nino
+      t.integer :rent_min
+      t.string :rent_max
```
Comment (arpanpal) [inline db/migrate/20181011035806_create_tenants.rb]: This should be an integer I think

Comment (arpanpal) [reply db/migrate/20181011035806_create_tenants.rb]: what is this integer field?

**Pull request merged**